wks-recruiter-interface/src/app/layout/tests/test-questions/test-questions.component.ts

```diff
@@ -65,6 +68,47 @@ export class TestQuestionsComponent implements OnInit {
     });
   }
 
+  setLanguages(fromLanguage, toLanguage) {
```

Comment:
W tej metodzie w ogóle niepotrzebne są argumenty, bo wywołujesz ją raz i przekazujesz this.<nazwa_zmiennej>.